modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java

```diff
@@ -990,6 +990,25 @@ public void testIsNotifiableUser() throws Exception {
 		Assert.assertTrue(
 			_workflowTaskManager.isNotifiableUser(
 				user.getUserId(), workflowTask.getWorkflowTaskId()));
+
+		PermissionChecker userPermissionChecker =
+			PermissionCheckerFactoryUtil.create(user);
+
+		PermissionThreadLocal.setPermissionChecker(userPermissionChecker);
```

Comment:
Hi @devpedrotavares, should we get the original value of the permission checker to restore it on the PermissionThreadLocal after we finish using it?

Reply:
Hi @linolaoj , Yes, I just changed it!